clients/admin-ui/src/features/dataset/EditCollectionDrawer.tsx

```diff
@@ -28,6 +28,15 @@ const EditCollectionDrawer = ({ collection, isOpen, onClose }: Props) => {
   const collectionIndex = useSelector(selectActiveCollectionIndex);
   const [updateDataset] = useUpdateDatasetMutation();
   const toast = useToast();
+  const deleteMessage = (
```

Comment:
Nit: I generally prefer keeping the jsx in the return block, even if its passed into a prop.